setup.py

```diff
@@ -2,7 +2,7 @@
 from setuptools import setup
 
 PROJECT_ROOT, _ = os.path.split(__file__)
-REVISION = '0.3.5'
+REVISION = '0.3.6'
```

Comment:
Please don't do that. Maintainers will increment version when release will be done.